setup.py

```diff
@@ -36,7 +36,7 @@
         "GitPython",
         "jinja2",
         "jsonschema",
-        "PyInquirer==1.0.2",
+        "questionary",
```

Comment:
You said that a feature we're using was in a recent PR? We should pin a minimum version here if we are using new features.

Reply:
Good point - I ended up not using it (validation support for checkboxes and aparrently lists).
But nevertheless it's probably a good idea to check compatibility of older versions, as it is frequently updated. Will have a look!

Reply:
Have set it to >= 1.8.0, which is the version where they implemented `unsafe_prompt`